package.json

```diff
@@ -65,6 +65,7 @@
         "cors": "2.8.5",
         "express": "4.17.1",
         "fast-glob": "3.3.0",
+        "gpt-3-encoder": "1.1.4",
```

Comment:
Not sure if this also supports `gpt3.5` or even `gpt4`

I would look into this: https://www.npmjs.com/package/@dqbd/tiktoken
It's a ported version of the `tiktokenizer` official library of OpenAI